### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by functionstackx**
Updated readme.md to include contact info, sponsors, about and screenshots

https://github.com/LemonFace0309/msmr/tree/readme

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+    <td>Ryerson University</td>
+     <td>Royal Bank of Canada</td>
+     <td>Canadian Psychological Association</td>
+  </tr>
+  <tr>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/Ryerson.jpg" width=200></td>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/RBC.png" width=200 ></td>
+    <td><img src="https://msmr-blue.vercel.app/images/sponsors/CPA.jpg" width=200 ></td>
+  </tr>
+ </table>
```
Comment (jimmyygit) [inline README.md:32]: isn't there Markdown syntax to make tables too?

**Review by jimmyygit — APPROVED** [review]
Sexc

File: README.md (hunk 1 above)
Comment (functionstackx) [reply README.md:32]: @jimmyygit i am not sure, can you provide me with some links to further explore?

Comment (jimmyygit) [reply README.md:32]: Some resources from google :)
https://www.markdownguide.org/extended-syntax/#tables
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]
can you merge into development first so we don't get separate heads for main and development later?

File: README.md (hunk 1 above)
Comment (functionstackx) [reply README.md:32]: @jimmyygit i am gonna just do like a nice image collage of all the sponsors in the future instead of using tables

**Review by LemonFace0309 — APPROVED** [review]

**Pull request merged**